src/lib/djerba/plugins/supplement/body/resources/WGS.description.versions.html

```diff
@@ -17,7 +17,7 @@
 In cases where OncoKB does not use MANE Select, links in annotation use the corresponding alteration in OncoKB.
 Copy number variations were called using <a href=${versions.PURPLE_LINK}>Purple</a> (${versions.PURPLE_VERSION}). 
 Microsatellite (MS) Instability status is called using <a href=${versions.MICROSATELLITE_LINK}>msisensor-pro</a> (${versions.MICROSATELLITE_VERSION}) and a custom list of MS sites created by msisensor-pro for the current reference genome.
-Hrdetect takes SNVs and In/Dels from mutect2. The proportion of deletions that are at microhomologous sites is summarized as "Microhomologous Deletions".
+Homologous recombination deficiency (HRD) status is called using HRDetect (Davies et al. 2017), a weighted logistic regression model, using the signature.tools.lib R package (Degasperi et al. 2020). HRDetect takes SNVs and in/dels from MuTect2. The proportion of deletions that are at microhomologous sites is summarized as "Microhomologous Deletions".
```

Comment:
Should we put a link to the paper here? We've done so elsewhere, eg. for PURPLE.